nxdrive/gui/folders_dialog.py

```diff
@@ -329,6 +335,40 @@ def _add_subgroup_duplicate_behavior(self, layout: QHBoxLayout, /) -> None:
         # Prevent previous objects to take the whole width, that does not render well for human eyes
         layout.addStretch(0)
 
+    def _new_folder_button_action(self) -> None:
+        """Update new_folder_button appearance and refresh button_ok state."""
+        if self.new_folder.isReadOnly():
+            self.new_folder.setReadOnly(False)
+            self.new_folder.setStyleSheet("")
+            self.new_folder_button.setText(Translator.get("OK"))
+        else:
+            self.new_folder.setReadOnly(True)
+            self.new_folder.setStyleSheet("* { background-color: rgba(0, 0, 0, 0); }")
+            self.new_folder_button.setText(Translator.get("EDIT"))
+
+        self.button_ok_state()
+
+    def _add_subgroup_new_folder(self, layout: QHBoxLayout, /) -> None:
+        """Add a sub-group for the new folder option."""
+        label = QLabel(Translator.get("NEW_REMOTE_FOLDER"))
+        label.setToolTip(Translator.get("NEW_REMOTE_FOLDER_TOOLTIP"))
+        label.setCursor(QCursor(QtCore.Qt.WhatsThisCursor))
+
+        self.new_folder = QLineEdit()
+        self.new_folder.setStyleSheet("* { background-color: rgba(0, 0, 0, 0); }")
+        self.new_folder.setReadOnly(True)
+        self.new_folder.setFrame(False)
+        expr = QtCore.QRegExp('^[^/:\\|*><?"]+')
```

Comment:
Is it possible to move those chars to a constant? Something like `INVALID_CHARS`.